framework/src/problems/EigenProblem.C

```diff
@@ -555,6 +555,10 @@ EigenProblem::solve()
     _nl->solve();
     _nl->update();
 
+    // with PJFNKMO solve type, we need to evaluate the linear objects to bring them up-to-date
+    if (solverParams()._eigen_solve_type == Moose::EST_PJFNKMO)
+      execute(EXEC_LINEAR);
+
```

Comment:
We need to use the flux to evaluate the power density, which is done through an aux kernel executed on linear. With PJFNK, we do not need this because we have residual evaluation at the end. Does this make sense to you?

Reply:
Will this execute all aux kernels? Or just the one needed for scaling the eigenvector?

Reply:
No, this only executes all aux kernels/postprocessors on linear and the postprocessor could be used for scaling (normalization). In that case, if we do not have this, we will do the normalization first with a garbage value of the postprocessor, and check if the postprocessor value becomes 1 with a call to `execute(EXEC_LINEAR)`, and we will get no and redo the scaling.